Makefile

```diff
@@ -161,6 +161,11 @@ proto:
 	@buf format -w
 .PHONY: proto
 
+typescript:
```

Comment:
I think we can include it into `make generate` commands

Reply:
https://github.com/zeta-chain/node/pull/825/commits/5397f6050c8a913040c1bbf494f1045d4267f27f